### DISCUSSION TITLE
잡코리아 교육부분에 학원수료내용을 적는게  좋을까요?

### DISCUSSION THREAD

**Discussion by gksgpwls0306** [biancasuji/biancasuji_#7] [Q&A] [answered]
잡코리아 교육부분에 학원수료내용을 적는게  좋을까요? 
과정은 웹인데 지원은 앱으로 하고 있어서 질문이 들어올 것 같은데 빼는게 좋을까요?

**Comment by biancasuji** [comment, marked as answer]
넣는게 좋습니다! :) 
이게 안적어도 공백으로 보여져서 질문은 들어올꺼에용! 👍
